qcodes/instrument_drivers/devices.py

```diff
@@ -57,9 +57,9 @@ def __init__(self,
             self.label = "{}_attenuated".format(self.v1.label)
 
         if name:
-            self.name = name
+            self._short_name = name
         else:
-            self.name = "{}_attenuated".format(self.v1.name)
```

Comment:
I think we can avoid this by just passing the name to the super method. There is no reason why this should be set here

Reply:
Thanks. Done